website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -591,6 +591,12 @@ Additionally, some implementations allow rsaPSS OID certificates to contain
 restrictions on signature parameters allowed by this certificate, but Go and
 Vault do not support adding such restrictions.
 
+At this time Go lacks support for generating CSRs that are PSS signed. If
```

Comment:
```suggestion
At this time Go lacks support for CSRs with the PSS signature algorithm. If
```

It lacks more than just generating support, also lacks verification and parsing (SPKI is left Raw and not parsed).

Reply:
Thanks, applied the fix in the PR feedback commit.